tibiapy/enums/tibia_enums.py

```diff
@@ -0,0 +1,11 @@
+from enum import Enum
+
+
+class Gender(Enum):
```

Comment:
CipSoft refers to this as `Sex` everywhere, including their [store product](https://tibia.fandom.com/wiki/Sex_Change), so I think this enum should be called `Sex`.

Also, we don't want to get into the `Sex = Gender` controversy 👀